### PR TITLE
Change dxTreeView expandEvent option type to String

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/tree_view/ui.tree_view.base.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ var TreeViewBase = HierarchicalCollectionWidget.inherit({
 
             /**
              * @name dxTreeViewOptions.expandEvent
-             * @type Enums.TreeViewExpandEvent
+             * @type String
```
Comment (San4es) [inline js/ui/tree_view/ui.tree_view.base.js]: String -> string

**Review by San4es — CHANGES_REQUESTED** [review]

**Review by San4es — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by TikhonovaTatyana — COMMENTED** [review]
Add @acceptValues

**Comment by San4es** [comment]
>Add @acceptValues

The `@acceptValues` attribute is obsolete. In fact, it is the same thing as `@enum` and `@type Enums.SomeEnumName`.

**Review by TikhonovaTatyana — CHANGES_REQUESTED** [review]
There should be acceptValues or Enum, if the option works only with click and dblclick

**Comment by San4es** [comment]
> if the option works **only with click and dblclick**

@mrjohn0011  @lukyanovas  what about other events: dxclick, dxhold, etc?

**Comment by mrjohn0011** [comment]
We (Me and @San4es) have decided to close this pull request because it is more convenient to use Enum with all accepted values than string.

**Pull request closed without merging**